.gitignore

```diff
@@ -30,3 +30,7 @@ docker-compose.override.yml
 
 server_config.yaml
 /graphs/
+
+.idea/
```

Comment:
these should exist in a global git ignore

Reply:
okay